en/drafts/originals/agent-based-model-communication-networks.md

```diff
@@ -555,22 +559,20 @@ plt.colorbar(label="Number of Agents present in Cell")
 
 ```
 
-{% include figure.html filename="en-or-agent-based-model-communication-networks-04.png" alt="This shows a color mesh, a 2-dimensional grid where each cell in the grid is colored based on how many agents are present on it, on a scale of 0 to 3, with lighter yellow colors indicating more agents and darker blue colors indicating less agents. The figure shows that some cells are more crowded than others, but the distribution does look rather random." caption="Figure 4. A color mesh showing how many agents are present on each cell of our grid space" %}
+{% include figure.html filename="en-or-agent-based-model-communication-networks-04.png" alt="Two-dimensional grid where each cell is colored based on how many agents are present on it, on a scale of 0 to 3, with lighter yellow colors indicating more agents and darker blue colors indicating fewer agents." caption="Figure 4. Color mesh showing the number of agents present on each cell of our grid space" %}
```

Comment:
I noticed that there is a small typo in the figure's title 'Number of Agents presents in Cell'.

Could you please send me a new figure with the corrected title, perhaps something like 'Number of Agents Present in Each Cell'? You can email it to me directly, publishing.assistant[@]programminghistorian.org

Reply:
@maltevogl

Reply:
I ll generate a new figure and send it to you.

Reply:
Thank you!